src/views/Home.jsx

```diff
@@ -1,18 +1,52 @@
 import { SingleList } from '../components/SingleList';
 import './Home.css';
+import { createList } from '../api/firebase';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export function Home({ data, setListPath, userId, userEmail }) {
+	const [name, setName] = useState('');
+	const navigate = useNavigate();
+
+	const handleSubmit = async (event) => {
+		event.preventDefault();
+		try {
+			const newList = await createList(userId, userEmail, name);
+			setListPath(newList);
+			setName('');
+			alert('The item has been added.');
+			navigate('/list');
+			localStorage.setItem('tcl-shopping-list-path', newList);
+		} catch (err) {
+			console.error(err);
+		}
```

Comment:
you need to alert the user if the item was not added, instead of only doing a console log.

Reply:
done

Reply:
done